datafusion/core/src/physical_plan/joins/cross_join.rs

```diff
@@ -434,15 +453,7 @@ impl CrossJoinStream {
                     Some(result)
                 }
                 other => {
-                    debug!(
```

Comment:
I think it is a nice improvement that the metrics are now included in things like `EXPLAIN ANALYZE`